app/src/organisms/ODD/CameraSettings/CameraControls/index.tsx

```diff
@@ -21,21 +24,30 @@ export type ActiveControlView =
 
 export interface CameraControlsProps {
   toggleShowControls: () => void
+  runId?: string
 }
 
 export function CameraControls({
   toggleShowControls,
+  runId,
 }: CameraControlsProps): JSX.Element {
   const { t } = useTranslation('device_settings')
   const [isLoading, setIsLoading] = useState(false)
 
   const [activeSubView, setActiveSubView] = useState<ActiveControlView>(null)
   const settings = useCameraSettingsValues()
-  const { createCameraImageSettings } = useCreateCameraImageSettings()
+  const globalMutation = useCreateCameraImageSettings()
```

Comment:
```suggestion
  const createGlobalCameraSettings = useCreateCameraImageSettings()
```
Kind of a nit, but I'd give this a more camera specific name, so people reading the code in the future don't have to do a double take to figure out what this variable refers to.